tests/test_models.py

```diff
@@ -29,12 +29,11 @@ def get_sample(model_class):
         smp.PSPNet,
         smp.UnetPlusPlus,
         smp.MAnet,
-        smp.UPerNet,
     ]:
         sample = torch.ones([1, 3, 64, 64])
-    elif model_class == smp.PAN:
+    elif model_class == smp.PAN or model_class == smp.UPerNet:
         sample = torch.ones([2, 3, 256, 256])
-    elif model_class == smp.DeepLabV3:
```

Comment:
Just curious why UPerNet requires a bigger input size.

Reply:
Hi @qubvel ,

Actually, UPerNet don't need to use a larger size for testing,
but PyTorch's nn.BatchNorm2d does not support inputs with shape (B, C, H, W) == (1, C, 1, 1).

## Why:
In UPerNet, the [PSPModule](https://github.com/qubvel-org/segmentation_models.pytorch/blob/8def40f3b35026fa5ebc4b14ddab2feeb0010111/segmentation_models_pytorch/decoders/upernet/decoder.py#L17-L30) uses AdaptiveAvgPool2d, which results in outputs of:
`[(B, C, 1, 1), (B, C, 2, 2), (B, C, 3, 3), (B, C, 6, 6)]`

When model.train() mode, the Batch Normalization (BN) layers are activated.
If the batch size is 1, it triggers the following error:
`raise ValueError(f"Expected more than 1 value per channel when training, got input size {size}")`

[PSPNet](https://github.com/qubvel-org/segmentation_models.pytorch/blob/8def40f3b35026fa5ebc4b14ddab2feeb0010111/segmentation_models_pytorch/decoders/pspnet/decoder.py#L11-L12) also encounters this issue, but in that case, it skips the BN layer for the 1x1 size directly.

## Solution:

- Increase the batch size to 2 for specific models:
```
elif model_class in [smp.PSPNet, smp.UPerNet]:
    sample = torch.ones([2, 3, 64, 64])
```

- Set the batch size to 2 uniformly for all models:
```
if model_class in [
    smp.Unet,
    smp.Linknet,
    smp.FPN,
    smp.PSPNet,
    smp.UnetPlusPlus,
    smp.MAnet,
    smp.UPerNet,
]:
    sample = torch.ones([2, 3, 64, 64])
```

Reply:
Thanks for clarifying! I suppose 
```
elif model_class in [smp.PSPNet, smp.UPerNet]:
    sample = torch.ones([2, 3, 64, 64])
```
Is better with a small comment in the code

Reply:
Looks good